src/table/virtualized-table/Table.tsx

```diff
@@ -17,24 +16,23 @@ import getHeights from './utils/get-height';
 import useScrollbarWidth from './hooks/use-scrollbar-width';
 
 const Table = (props: TableProps) => {
-  const { rect, pageInfo, paginationNeeded } = props;
+  const { rect, pageInfo, tableData } = props;
+  const { totalsPosition, columns, paginationNeeded } = tableData;
```

Comment:
I think you can combine these two lines. because it's already passed from your check at `index.js` and you are 100% sure that `totalsPosition` is returning an object of `{ atTop: boolean, atBottom: boolean }` at any case, so my suggestion would be: 
```jsx
const { rect, pageInfo, tableData: { totalsPosition, columns, paginationNeeded } } = props;
```

Reply:
I can but it would end up being 5 lines of code instead of 2 thanks to prettier so I will not change that